spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -1582,8 +1604,10 @@ public void testPartitionsTableLastUpdatedSnapshot() {
             .set("total_data_file_size_in_bytes", dataFiles.get(0).fileSizeInBytes())
             .set("position_delete_record_count", 0L)
             .set("position_delete_file_count", 0)
+            .set("total_position_delete_file_size_in_bytes", 0L)
             .set("equality_delete_record_count", 0L)
             .set("equality_delete_file_count", 0)
+            .set("total_equality_delete_file_size_in_bytes", 0L)
```

Comment:
Do we need this line and L1607 in 3.4 and 3.5?

Reply:
done